pkgs/development/libraries/totem-pl-parser/default.nix

```diff
@@ -17,7 +17,11 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ meson ninja pkg-config gettext gobject-introspection ];
-  buildInputs = [ libxml2 ];
+  buildInputs = [ libxml2 glib ];
```

Comment:
Is this only required for cross? Can we set strictDeps?

Reply:
I've set strictDeps and updated accordingly.